api/app/settings/common.py

```diff
@@ -715,6 +716,18 @@
     "USER_THROTTLE_CACHE_OPTIONS", default={}
 )
 
+ONBOARDING_REQUEST_THROTTLE_CACHE_NAME = "onboarding-request-throttle"
+ONBOARDING_REQUEST_THROTTLE_CACHE_BACKEND = env.str(
+    "ONBOARDING_REQUEST_THROTTLE_CACHE_BACKEND",
+    "django.core.cache.backends.db.DatabaseCache",
```

Comment:
Using the DB backend as the default here so we can deploy it before testing Redis in production